delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/model/DigestAlgorithm.scala

```diff
@@ -10,22 +9,32 @@ import scala.util.Try
 /**
   * A digest algorithm
   */
-final case class DigestAlgorithm private (value: String) extends AnyVal {
+final case class DigestAlgorithm private (value: String, digest: MessageDigest) {
```

Comment:
We serialize the DigestAlgorithm somewhere no ?
It is better not add the MessageDigest directly here but just to add a def below no ?